text/0186-per-tracer-configuration.md

```diff
@@ -0,0 +1,124 @@
+# Per Tracer Configuration
+
+Support setting configuration (Sampler, IdGenerator, Span Limits, etc) on
```

Comment:
Could you give an exhaustive list of configuration to be shared? Or is it any kind of configuration, including processors exporters, and resources?

Reply:
It is any configuration. But I don't believe that includes the resource? The resource is part of the Provider but it isn't a configuration value passed by the user when building a Provider, right?

I used `etc` because it should cover any future additional configuration added to the Tracer Provider.

I left out explicitly including Span Processors because it is a bit of a pain point I cover later in the OTEP, but I could add it back to the list just to be complete.

And the Exporter is configured as part of the Processor so it is based on if those are configurable or not.